jsx-runtime/src/index.js

```diff
@@ -43,7 +43,7 @@ function createVNode(type, props, key, __source, __self) {
 		key,
 		ref,
 		constructor: undefined,
-		_vnodeId: --vnodeId,
+		_vnodeId: (vnodeId = (vnodeId - 1) | 0),
```

Comment:
Note using [bitwise "or" operator restricts `vnodeId` to a 32-bit int](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR) (±2\**31) whereas the [safe integer is ±2**53](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger). I wonder what the tradeoff is between doing `| 0` vs `% 2**53-1` (precalcuating and storing `2**53-1`).